Makefile

```diff
@@ -43,4 +43,5 @@ $(ASOBJFILE): %.o: %.s
 	$(AS) $(ASFLAGS) -o $@ $<
 	
 $(COBJFILE): %.o: %.c
-	$(CPP) $(CPPFLAGS) $< | $(CC1) $(CFLAGS) | $(AS) $(ASFLAGS) -o $@ 
\ No newline at end of file
+	$(CPP) $(CPPFLAGS) $< | $(CC1) $(CFLAGS) -o $*.i
```

Comment:
.i file extension is for preproccesed C files in pret repos. export directly to .s